### ISSUE TITLE
requirements.txt and python version

### ISSUE THREAD

**Issue by zerafachris** [leonard-seydoux/scatnet#3]
Hi, is it possible to provide a requirements.txt and python version for this code?

**Comment by Erik-BM** [comment]
@zerafachris this repo seems to be dead. 

So if you are interested in a TF2.0 version, I've implemented it here: https://github.com/NorwegianSeismicArray/NORSARAISystem/blob/4182291dac39c0ce61f36e71d828438721bde52a/nais/Models.py#L729
See for example usage: https://github.com/NorwegianSeismicArray/NORSARAISystem/blob/main/examples/scatnet.py

**Comment by seismoprocess** [comment]
@Erik-BM It would be very helpful if you could share a demo example for the three component seismogram clustering.

**Comment by Erik-BM** [comment]
@seismoprocess yes, I'm planning to work on that. However, we've found that actually learning the wavelets is more computationally expensive that the benefits. Generating features from static wavelets is >4000x faster than learning them. 
Therefore, I would recommend looking at https://github.com/scatseisnet/scatseisnet/tree/api . They have notebooks containing the entire pipeline.

**Comment by leonard-seydoux** [comment]
Hi guys, and sorry for the inaction from my side, I indeed dropped this project since the deprecation of TensorFlow 1 and didn't receive notifications from this conv. As @Erik-BM mentioned, we deployed the new repository at https://github.com/scatseisnet/scatseisnet and will try to maintain it as far as possible. I know other groups that are also going to provide help with the learnable version. Indeed, it's extremely fast to generate the features from the static version, which is partly why we decided to keep this version. 

I will look into your version @Erik-BM!
